load.php

```diff
@@ -169,29 +204,52 @@ function perflab_render_generator() {
 add_action( 'wp_head', 'perflab_render_generator' );
 
 /**
- * Loads the active performance modules.
+ * Checks whether the given module can be loaded in the current environment.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $module Slug of the module.
+ * @return bool Whether the module can be loaded or not.
+ */
+function perflab_can_load_module( $module ) {
+	$module_load_file = plugin_dir_path( __FILE__ ) . 'modules/' . $module . '/can-load.php';
+
+	// If the `can-load.php` file does not exist, assume the module can be loaded.
+	if ( ! file_exists( $module_load_file ) ) {
+		return true;
+	}
+
+	// Require the file to get the closure for whether the module can load.
+	$can_load = require $module_load_file;
+
+	// If the `can-load.php` file is invalid and does not return a closure, assume the module can be loaded.
+	if ( ! is_callable( $can_load ) ) {
+		return true;
+	}
+
+	// Call the closure to determine whether the module can be loaded.
+	return $can_load();
+}
+
+/**
+ * Loads the active and valid performance modules.
  *
  * @since 1.0.0
```

Comment:
Maybe let's add a doc-block since the rename here is somewhat significant.
```suggestion
 * @since 1.0.0
 * @since n.e.x.t Renamed to perflab_load_active_and_valid_modules().
```

Reply:
Doc-block added for function rename.